lib/process-services/src/lib/process-list/components/start-process.component.spec.ts

```diff
@@ -747,4 +709,107 @@ describe('StartFormComponent', () => {
             expect(processNameInput.disabled).toEqual(false);
         });
    });
+
+    describe('Empty Template', () => {
+
+        it('should show no process definition available template when application/process definitions are empty', async() => {
+            getDeployedApplicationsSpy = spyOn(appsProcessService, 'getDeployedApplications').and.returnValue(of([]));
+            getDefinitionsSpy.and.returnValue(of([]));
```

Comment:
https://w3.alfresco.com/confluence/pages/viewpage.action?spaceKey=AP&title=Code+Style+E2E  Style 05-14
 to dense

Reply:
I will go through the above confluence page, and next time will follow guide lines. Thanks :+1: